src/main/java/Greet.java

```diff
@@ -0,0 +1,87 @@
+import java.util.Scanner;
+
+
+
+public class Greet {
+
+    private static Task[] list= new Task[100];
```

Comment:
Maybe you can convert the number 100 into a named constant for better understanding. I suggest naming it as MAX_TASK_SIZE